mlflow/ml-package-versions.yml

```diff
@@ -315,5 +315,12 @@ spark:
     maximum: "3.1.2"
     requirements: ["scikit-learn"]
     run: |
-      # TODO: Add datasource autologging tests
       find tests/spark_autologging/ml -name 'test*.py' | xargs -L 1 pytest --large
+
+      # Spark datasource autologging currently doesn't work for public pyspark versions
```

Comment:
Can we print a warning here about that? Are we sure that it doesn't work with PySpark 3.1.2?

Reply:
> Can we print a warning here about that?
Sure!

> Are we sure that it doesn't work with PySpark 3.1.2?

Yes, I re-confirmed in this run: https://github.com/mlflow/mlflow/runs/3391728130